db_stress_tool/db_stress_test_base.cc

```diff
@@ -512,6 +512,14 @@ void StressTest::OperateDb(ThreadState* thread) {
 
       MaybeClearOneColumnFamily(thread);
 
+      if (FLAGS_sync_wal_one_in > 0 &&
+          thread->rand.Uniform(FLAGS_sync_wal_one_in) == 0) {
+        Status s = db_->SyncWAL();
+        if (!s.ok()) {
+          fprintf(stdout, "Unable to SyncWAL(): %s\n", s.ToString().c_str());
```

Comment:
We treat it not as a failure because it may not be supported?

Reply:
Right. I will treat it as an error when `!s.ok() && !s.IsNotSupported()`.